src/app/active-mode/Views/current-activity/VendorBounties.tsx

```diff
@@ -39,9 +40,34 @@ function mapStateToProps(state: RootState): StoreProps {
   };
 }
 
-type Props = ProvidedProps & StoreProps;
+function bountiesForActivity(
+  defs: D2ManifestDefinitions,
+  bounties: VendorItem[],
+  activity: DestinyActivityDefinition
+) {
+  return bounties.filter(({ item }) => {
+    const info = item?.hash && pursuitsInfo[item?.hash];
+    if (!info) {
+      return false;
+    }
+
+    for (const key in info) {
+      switch (key) {
+        case 'Destination':
+          return info[key].includes(activity.destinationHash);
+        case 'ActivityMode':
+          // eslint-disable-next-line no-extra-boolean-cast
+          return Boolean(activity.activityModeHashes)
+            ? activity.activityModeHashes.some((modeHash) => info[key].includes(modeHash))
```

Comment:
These are specifically `ActivityMode`, not `Activity` - you need to load a `DestinyActivityModeDefinition`. e.g. https://data.destinysets.com/i/ActivityMode:1264443021 

You can walk up the tree with `parentHashes` to find all applicable activity modes.

Reply:
awesome, was struggling with that and the destination / place hashes. I'll take a look by tonight to line everything up and then this should be good to go.

Reply:
should be set now, thanks for that!